Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -762,6 +763,100 @@ public IEnumerator TestInteractableToggleCollectionIndexUpdate()
             GameObject.Destroy(interactableToggleCollection.gameObject);
         }
 
+
+        /// <summary>
+        /// Test if Button state is reset when it goes out of focus from a pressed state
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestButtonStateResetWhenFocusLostAfterPinch()
+        {
+
+            Interactable interactable;
+            Transform interactableTransform;
+
+            InstantiatePressableButtonHoloLens2(
+                new Vector3(0.0f, 0.1f, 0.4f),
+                DefaultRotationToggle,
+                out interactable,
+                out interactableTransform);
+
+            Assert.True(interactable.IsEnabled);
+
+            var rightHand = new TestHand(Handedness.Right);
+            Vector3 focusPosition = new Vector3(0.015f, 0.015f, 0.3f);
+            Vector3 releaseDelta = new Vector3(0.05f, 0, 0);
+
+
+            // Initial Button State On Focus from Far Interaction Pointer
```

Comment:
Nit: Maybe something like

// Focus the hand on the button using the far ray pointer.